service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -148,6 +163,21 @@ public String getMostRecentUploadedOrderDocumentUrl(final List<Element<Generated
             .getDocument().getBinaryUrl();
     }
 
+    public CaseDetails removeOrderProperties(CaseDetails caseDetails, GeneratedOrderType orderType) {
+        if (orderType.equals(EMERGENCY_PROTECTION_ORDER)) {
```

Comment:
No need to pass `orderType` here. `.remove(...)` doesn't care if the map contains the key or not, if it doesn't it just won't alter the map